xml/signals/NYCS-1937/aspects.xml

```diff
@@ -197,7 +197,7 @@
     </aspect>
 
     <aspect>
-      <name>Stop; Manual Block</name><!-- to avoid duplicating Stop below -->
```

Comment:
Can we put a `<comment>` element in here to carry the content of that XML comment?

Reply:
I have added the option to have an attribute `comment` to this tag. It made it easier to keep the comment together with the item it comments on.